### PR TITLE
NFDIV-2587

### PR DESCRIPTION
**Pull request by tomxelliott**
### Change description ###
Configure D84 offline document verified

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/NFDIV-2587

Screenshot 2022-08-09 at 15 16 09

Screenshot 2022-08-09 at 15 17 41

Screenshot 2022-08-09 at 15 28 32

### CODE REVIEW

File: src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    "applicant1Gender": "female",
+    "applicant1ContactDetailsType": "public",
+    "applicant1FinancialOrder": "No",
+    "applicant1Offline": "Yes",
+    "applicant2Offline": "Yes",
+    "solServiceTruthStatement": "I believe that the facts stated in the application are true.",
+    "coScannedD84Form": {
+      "document_url": "http://localhost:8080/f62d42fd-a5f0-43ff-874b-d1666c1bf00d",
+      "document_filename": "doc1.pdf",
+      "document_binary_url": "http://localhost:8080/f62d42fd-a5f0-43ff-874b-d1666c1bf00d/binary"
```
Comment (amoriartyMOJ) [inline src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json:19]: Does it matter if we set the url to localhost for these type of requests? Will these work fine in AAT etc :)?

File: src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/AppliedForCoPrinter.java [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    }
+
+    private Map<String, Object> templateContent(final CaseData caseData,
+                                                final Long caseId,
+                                                final Applicant applicant) {
+
+        final Map<String, Object> templateContent = new HashMap<>();
+
+        templateContent.put(CASE_REFERENCE, formatId(caseId));
+        templateContent.put(NAME, join(" ", applicant.getFirstName(), applicant.getLastName()));
```
Comment (amoriartyMOJ) [inline src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/AppliedForCoPrinter.java:105]: picky but Applicant has a getFullName method which could be used?

Comment (tomxelliott) [reply src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/AppliedForCoPrinter.java:105]: I don't want the middle names on this though

**Review by amoriartyMOJ — APPROVED** [review]

File: src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json (hunk 1 above)
Comment (tomxelliott) [reply src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json:19]: Doesn't impact as works fine in Preview

Comment (amoriartyMOJ) [reply src/functionalTest/resources/responses/response-caseworker-offline-document-verified-d84-about-to-submit.json:19]: Thanks! approved 👍

**Pull request merged**